blockchain/x/liquidator/keeper.go

```diff
@@ -0,0 +1,85 @@
+package liquidator
+
+// --------- Keeper ----------
+type Keeper struct {
+	// CDP keeper
+	// auction keeper
+}
+
+const moduleAccountAddress = cdp.LiquidationAccountAddress
+
+func (k Keeper) StartCollateralAuction(seizedCdpId uint64) { // aka Cat.flip
+	// get Seized CDP (fom ID)
+	// calculate how much dai to try and raise in this auction	// daiToRaise := min(seizedCdp.Debt, params.maxDaiToBeRaised)
+	// calculate how much collateral to sell (based on above)
+	// subtract these values from the seizedCDP
+	// start "forward reverse" auction type // auctionKeeper.StartForwardReverseAuction(moduleAccountAddress, lot, daiToRaise, seizedCDP.OriginalOwner)
+	// store seizedCDP
+	// result: dai in transferred to moduleAccount, collateral is transferred from module account to buyer, (and any excess collateral is transferred to original CDP owner)
+}
+
+func (k Keeper) StartDebtAuction() { // aka Vow.flop
+	// check there is enough seized debt to be "sold"
```

Comment:
What does 'enough' mean in these statements?

Reply:
mcd runs auctions in chunks of a predetermined size. So here it would be running an auction to raise a fixed amount of stable coin. So there needs to be at least that much seized debt